roboflow/util/prediction.py

```diff
@@ -83,6 +84,15 @@ def plot_annotation(axes, prediction=None, stroke=1):
             + " | Confidence: "
             + str(prediction["confidence"])
         )
+    elif prediction["prediction_type"] == INSTANCE_SEGMENTATION_MODEL:
+        points = [[p["x"], p["y"]] for p in prediction["points"]]
+        polygon = patches.Polygon(
+            points,
+            linewidth=stroke,
+            edgecolor="r",
+            facecolor="none",
+        )
+        axes.add_patch(polygon)
```

Comment:
i was not exactly sure how to test this method, but this _seems_ right?

Reply:
Seems good on vis on my side